mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/utils/TelemetryUtils.java

```diff
@@ -168,4 +175,37 @@ public static Boolean getConnectedToWifi(Context context) {
     return isConnectedToWifi;
   }
 
+  /**
+   * Returns the screen backlight brightness between 0 and 100, -1 if the check failed.
+   */
+  public static int getScreenBrightness(Context context) {
+    int screenBrightness;
+    try {
+      screenBrightness = android.provider.Settings.System.getInt(
+          context.getContentResolver(),
+          android.provider.Settings.System.SCREEN_BRIGHTNESS);
+
+      // Android returns values between 0 and 255, here we normalize to 0-100.
+      screenBrightness = (int) Math.floor(100.0 * screenBrightness / 255.0);
+    } catch (Settings.SettingNotFoundException exception) {
+      screenBrightness = -1;
+    }
+
+    return screenBrightness;
+  }
+
+  /**
+   * Returns the current system volume level
+   */
+  public static int getVolumeLevel(Context context) {
+    AudioManager audioManager = (AudioManager) context.getSystemService(Context.AUDIO_SERVICE);
+    return audioManager.getStreamVolume(AudioManager.STREAM_SYSTEM);
```

Comment:
Also, let's make sure to normalize this as an int from 0-100 (it looks like there's a [`getStreamMaxVolume(int)`](https://developer.android.com/reference/android/media/AudioManager.html#getStreamMaxVolume(int)) to normalize against)